cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,66 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 2.6.3 <http://docs.cask.co/cdap/2.6.3/index.html>`__
+=============================================================
+
+CDAP Bug Fixes
+--------------
+
+- Replace use of CDAP transaction snapshot codecs with the Tephra versions instead
+  (`CDAP-2496 <https://issues.cask.co/browse/CDAP-2496>`__)
+  
+  
+Upgrade Instructions
+--------------------
+
+- **Update Memory Setting for Transaction Service before Upgrading**
```

Comment:
This section was recommended in the case of one very large transaction snapshot file.  Not sure if this should be a general recommendation for us.  This is only necessary if the transaction snapshot that needs to be loaded is very substantial in size (100's of MB).